lib/bgs/dependents.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+module BGS
+  class Dependents < Service
+    include Helpers::Formatters
+
+    def initialize(proc_id:, payload:, user:)
+      @proc_id = proc_id
+      @payload = payload
+      @dependents = []
+      @dependents_application = @payload['dependents_application']
+
+      super(user)
+    end
+
+    # rubocop:disable Metrics/PerceivedComplexity
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def create
+      report_674 if @payload['report674']
+      add_children if @payload['add_child']
+      report_deaths if @payload['report_death']
+      report_divorce if @payload['report_divorce']
+      report_stepchild if @payload['report_stepchild_not_in_household']
+      report_child_event('child_marriage') if @payload['report_marriage_of_child_under18']
+      report_child_event('not_attending_school') if @payload['report_child18_or_older_is_not_attending_school']
+
+      @dependents
+    end
+
+    # rubocop:enable Metrics/CyclomaticComplexity
+    # rubocop:enable Metrics/PerceivedComplexity
+
+    private
+
+    def add_children
+      @dependents_application['children_to_add'].each do |child_info|
```

Comment:
Is it at all possible for `@dependents_application['children_to_add']` to be `nil`? If so, is there any way to safely recover?